common/src/main/java/com/sequenceiq/cloudbreak/monitoring/MonitoringEnablementCache.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.cloudbreak.monitoring;
+
+import java.util.concurrent.TimeUnit;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.cache.common.AbstractCacheDefinition;
+
+@Service
+public class MonitoringEnablementCache extends AbstractCacheDefinition {
+
+    private static final long MAX_ENTRIES = 5000L;
+
+    @Value("${monitoring.enablement.cache.ttl:1}")
```

Comment:
this property can go into common-config.yml -> e.g. under telemetry.monitoring.<whatever>, or even under status-processor config (not related strongly, but if that is disabled this won't be used)